components/Article/EditLink/EditLink.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { LocaleProvider } from './../../../providers/localeProvider';
+import EditLink from './index';
+import {
+  exampleRelativePath,
+  i18nMockDataEnglish,
+  i18nMockDataNonEnglish,
+} from './mockDataConstants';
+
+export default { component: EditLink };
+
+export const Edit = {
+  args: {
+    relativePath: exampleRelativePath,
+  },
+  render: (args: any) => {
```

Comment:
Can we use a decorator instead of render? https://storybook.js.org/docs/react/writing-stories/decorators